### ISSUE TITLE
Crash During Recording

### ISSUE THREAD

**Issue by cliss** [cliss/accidentalbot#44]
```
2019-02-07T03:18:04.872308+00:00 app[web.1]: /app/node_modules/irc/lib/irc.js:849
2019-02-07T03:18:04.872320+00:00 app[web.1]: throw err;
2019-02-07T03:18:04.872322+00:00 app[web.1]: ^
2019-02-07T03:18:04.872324+00:00 app[web.1]: 
2019-02-07T03:18:04.872326+00:00 app[web.1]: TypeError: Cannot read property 'filter' of undefined
2019-02-07T03:18:04.872329+00:00 app[web.1]: at chanModes (/app/node_modules/irc/lib/irc.js:300:37)
2019-02-07T03:18:04.872331+00:00 app[web.1]: at /app/node_modules/irc/lib/irc.js:319:25
2019-02-07T03:18:04.872333+00:00 app[web.1]: at Array.forEach (native)
2019-02-07T03:18:04.872334+00:00 app[web.1]: at Client.<anonymous> (/app/node_modules/irc/lib/irc.js:270:26)
2019-02-07T03:18:04.872336+00:00 app[web.1]: at emitOne (events.js:77:13)
2019-02-07T03:18:04.872338+00:00 app[web.1]: at Client.emit (events.js:169:7)
2019-02-07T03:18:04.872339+00:00 app[web.1]: at iterator (/app/node_modules/irc/lib/irc.js:846:26)
2019-02-07T03:18:04.872341+00:00 app[web.1]: at Array.forEach (native)
2019-02-07T03:18:04.872342+00:00 app[web.1]: at Socket.handleData (/app/node_modules/irc/lib/irc.js:841:15)
2019-02-07T03:18:04.872344+00:00 app[web.1]: at emitOne (events.js:77:13)
2019-02-07T03:18:04.872345+00:00 app[web.1]: at Socket.emit (events.js:169:7)```

**Comment by cliss** [comment]
Perhaps consider updating/upgrading [`irc`](https://github.com/martynsmith/node-irc)? Currently on 0.3.6; latest is [0.5.2](https://github.com/martynsmith/node-irc/releases/tag/v0.5.2)

**Comment by cliss** [comment]
This was triggered once by me doing `/mode #atpfm -q $~a`

**Comment by cliss** [comment]
Possible option [here](https://github.com/martynsmith/node-irc/issues/514)

**Comment by cliss** [comment]
Fixed in 6a650dc4e6f5eeec1f11c200fa4a803daa42c5aa
